src/fundus/publishers/de/dw.py

```diff
@@ -13,11 +13,14 @@
     generic_date_parsing,
     generic_text_extraction_with_css,
     generic_topic_parsing,
+    strip_nodes_to_text,
 )
 
 
 class DWParser(ParserProxy):
     class V2(BaseParser):
+        VALID_UNTIL = datetime.date(2024, 1, 18)
+
         _paragraph_selector = CSSSelector("div.rich-text > p")
```

Comment:
They also sometimes mention an update of the article at the end, this should be able to be handled easily since it's in italics: https://www.dw.com/de/esc-2024-schwedische-k%C3%BCnstler-gegen-teilnahme-israels/a-68011968

Reply:
> DW seems to sometimes have additional tags at the end of the article

Are you talking about this `jj/sti (dpa, afp, rtr)`?
I can only think of regular expressions. But I don't know how to get them working in `XPath`.
Anyways, that would be a regular expression matching the above string.
`.*\((rtr, |dpa, |afp, ){0,2}(rtr|dpa|afp)\)$`

> They also sometimes mention an update of the article at the end, this should be able to be handled easily since it's in italics

I added it to the selector.

Reply:
Yes, although it's not restricted to those: here are some that I found: `AR/jj (dpa, rtr, ap)` `kle/jj (kna, dpa, rtr, afp)` `pg/AR/kle (dpa, afp)` `pg/AR/haz (afp, dpa)` `kle/haz (dpa, rtr, afp)` not really sure what it's limited to though

Reply:
So unless we can use reg exp in XPath there is nothing much to do about it anyways, but updating the parser would be very important because it currently does not run.